src/binding.cc

```diff
@@ -57,9 +56,15 @@ static_assert(sizeof(v8::PromiseRejectMessage) == sizeof(size_t) * 3,
 
 static_assert(sizeof(v8::Locker) == sizeof(size_t) * 2, "Locker size mismatch");
 
-static_assert(sizeof(v8::ScriptCompiler::Source) ==
-                  align_to<size_t>(sizeof(size_t) * 9 + sizeof(int) * 2),
-              "Source size mismatch");
+static_assert(sizeof(v8::ScriptCompiler::CompilationDetails) ==
```

Comment:
32-bit build fails here:

```cpp
   ../../../src/binding.cc:59:15: error: static assertion failed due to requirement 'sizeof(v8::ScriptCompiler::CompilationDetails) == sizeof(unsigned int) * 3': CompilationDetails size mismatch
      59 | static_assert(sizeof(v8::ScriptCompiler::CompilationDetails) ==
         |               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      60 |                   sizeof(size_t) * 3,
         |                   ~~~~~~~~~~~~~~~~~~
   ../../../src/binding.cc:59:62: note: expression evaluates to '20 == 12'
      59 | static_assert(sizeof(v8::ScriptCompiler::CompilationDetails) ==
         |               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~
      60 |                   sizeof(size_t) * 3,
         |                   ~~~~~~~~~~~~~~~~~~
   1 error generated.
```

example of full log here:
 
https://build.opensuse.org/package/live_build_log/devel:languages:javascript/rusty_v8/openSUSE_Tumbleweed/i586